content/articles/changing-account-information.md

```diff
@@ -16,26 +16,44 @@ categories:
 
 ## Changing account email
 
-To change your account email, follow the steps described in the article [Changing the account email](/articles/changing-email#changing-the-account-email).
+<info>
+When you sign up as a user in our system, we create an account bound to that user. This is the Primary Account, and the email address is the same as the username. If you change the email address on the Account page, the account email address will be updated. **If you are not the primary account holder, and want to change your user email, follow the instructions in [Changing the user email](/articles/changing-email/#changing-the-user-email).**
+</info>
+
+<div class="section-steps" markdown="1">
+1. Go to [your account page](https://dnsimple.com/account).
+1. Click <label>Edit</label> at the bottom right of the <label>Account</label>card: 
+
+    ![Settings menu](/files/account-menu.png)
+
+1. Change the email, and click <label>Update Account</label>.
+
+    ![Email link](/files/account-email.png)
+
+</div>
 
 ## Changing other account data
 
 <div class="section-steps" markdown="1">
 1. Go to [your account page](https://dnsimple.com/account).
-1. On the account section, click the <label>Edit</label> link:
+1.  Click <label>Edit</label> at the bottom right of the <label>Account</label>card:
 
     ![Access to the account information section](/files/account-edit.png)
 
-1. Change or set the information according to your needs:
+1. Change or set the information according to your needs.
+
+   You can edit your account name, email, country of operation, and change your answers to any of the security screening questions. 
 
    ![Account information form](/files/account-information-form.png)
 
    <info>
    DNSimple requests this information, in compliance with applicable laws and regulations, to guarantee the security of your account.
```

Comment:
This is part of the Patriot Act, https://en.wikipedia.org/wiki/Know_your_customer and is just due diligence for this kind of business relationship. I don't think anyone's getting confused about whether or not they fall into these categories, but we can link the wikipedia article for the curious if that would be beneficial.